pegasus/sites.v3/code.org/public/teach-draft.haml

```diff
@@ -0,0 +1,183 @@
+---
+title: Teach Computer Science
+theme: responsive_full_width
+---
+
+%link{href: '/css/generated/design-system-pegasus.css', rel: 'stylesheet'}
+%link{href: "/css/generated/teach-styles.css", rel: "stylesheet"}
+
+-# FontAwesome icons
+- icon_party_horn = "fa-solid fa-party-horn"
+- icon_lightbulb_on = "fa-solid fa-lightbulb-on"
+- icon_link = "fa-solid fa-link"
+
+%section.top
+  .wrapper
+    .text-wrapper.col-50
+      %h1=hoc_s(:teach_page_top_heading)
+      %p=hoc_s(:teach_page_top_desc_01)
+      %p.body-two
+        =hoc_s(:teach_page_top_desc_02)
+      %a{href: CDO.studio_url("/users/sign_up"), class: "link-button"}
+        =hoc_s(:call_to_action_sign_up_free)
+    %figure.col-45{style: "float: right; margin-top: 1.25rem;"}
+      %img.rounded-corners.box-shadow-primary{src: "/images/teach-page-top.png", alt: "", style: "position: relative; width: 100%;"}
+    .clear
+
+%section.why-teach-cs.bg-neutral-light
+  .wrapper
+    %h2.centered
+      =hoc_s(:teach_page_why_teach_cs_heading)
+    %div.grid-container
+      %div.flex-container
+        %figure
+          %img{src: "/images/teach-page-why-cs-01.png", alt: ""}
+        .text-wrapper
+          %h3.heading-sm
+            =hoc_s(:teach_page_why_teach_cs_block_heading_01)
+          %p=hoc_s(:teach_page_why_teach_cs_block_desc_01)
+      %div.flex-container
+        %figure
+          %img{src: "/images/teach-page-why-cs-02.png", alt: ""}
+        .text-wrapper
+          %h3.heading-sm
+            =hoc_s(:teach_page_why_teach_cs_block_heading_02)
+          %p=hoc_s(:teach_page_why_teach_cs_block_desc_02)
+      %div.flex-container
+        %figure
+          %img{src: "/images/teach-page-why-cs-03.png", alt: ""}
+        .text-wrapper
+          %h3.heading-sm
+            =hoc_s(:teach_page_why_teach_cs_block_heading_03)
+          %p=hoc_s(:teach_page_why_teach_cs_block_desc_03)
+      %div.flex-container
+        %figure
+          %img{src: "/images/teach-page-why-cs-04.png", alt: ""}
+        .text-wrapper
+          %h3.heading-sm
+            =hoc_s(:teach_page_why_teach_cs_block_heading_04)
+          %p=hoc_s(:teach_page_why_teach_cs_block_desc_04)
+          %a{href: "/ai"}
+            =hoc_s(:call_to_action_explore_ai_resources)
+    %hr.dark
+    %div.centered
+      %h4=hoc_s(:teach_page_why_teach_cs_heading_02)
+      %a.link-button{href: CDO.studio_url("/catalog")}
+        =hoc_s(:call_to_action_explore_curriculum_catalog)
+
+%section.quote.bg-neutral-dark
+  .wrapper
+    %div.quotes-wrapper
+      %p.quote-text
+        =hoc_s(:teach_page_quote)
+      %p.no-margin-bottom Kevin Barry <span>•</span> CS Discoveries Teacher
```

Comment:
Is it okay that this "CS Discoveries Teacher" is hard-coded? Or I guess more what I'm getting at is if this is translatable?

Reply:
Good catch! I'll make this translatable 👍